src/account/MyAccount/RemoveAccount.tsx

```diff
@@ -0,0 +1,30 @@
+import "./scss/RemoveAccount.scss";
+
+import React, { useCallback, useState } from "react";
+import Card from "../Card";
+import CardHeader from "../CardHeader";
+import RemoveAccountModal from "./RemoveAccountModal";
+
+const RemoveAccount: React.FC = () => {
+  const [isVisibleModal, setModalVisibility] = useState(false);
+  const showModal = useCallback(() => setModalVisibility(true), []);
+  const hideModal = useCallback(() => setModalVisibility(false), []);
+
+  const header = <CardHeader title="Remove Account" />;
```

Comment:
Wouldn't that be better to say "Delete Account"?

"Remove" sounds like the action is revertible.